src/server/replica.h

```diff
@@ -28,6 +28,13 @@ class ConnectionContext;
 class JournalExecutor;
 struct JournalReader;
 
+struct ReplicaRoleInfo {
```

Comment:
The struct is used to return the data from GetReplicasRoleInfo. I dont think this is the right place for this struct.
This is the data master holds on its replica. 
Can we move it back to dflycmd.h ?

Reply:
I need it to be defined in server_family.h when I include it in metrics though

Reply:
Tried to move it to server_family.h (slightly better, I think)